ethereum-cpp-on-ubuntu/azuredeploy.parameters.json

```diff
@@ -0,0 +1,21 @@
+{
+  "$schema": "http://schema.management.azure.com/schemas/2015-01-01/deploymentParameters.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "newStorageAccountName": {
+      "value": "#####"
```

Comment:
We just enabled a new CI system and the ability to input strings instead ##### (we got feedback not everyone might understand what this means). See here how to set the parameter file for CI deployment. 
https://github.com/Azure/azure-quickstart-templates/tree/master/1-CONTRIBUTION-GUIDE#travis-ci